### PR TITLE
fix #142 can not only specify url as src

### LINKED ISSUE #142: can not only specify url as src
When I specify src as url like this:

```js
files: {
  'test.css':
    ['http://xxx/xx']
}
```

I got following exception:

**Warning: Destination (test.css) not written because src files were empty.**

I think in uncss.js:

```js
if ( src.length === 0 ) {
    grunt.fail.warn( 'Destination (' + f.dest + ') not written because src files were empty.' );
}

f.orig.src.forEach(function (source) {
    if (/^https?/.test(source)) {
        src.push(source);
        options.urls.push(source);
    }
});
```

It should be:

```js
f.orig.src.forEach(function (source) {
    if (/^https?/.test(source)) {
        src.push(source);
        options.urls.push(source);
    }
});
if ( src.length === 0 ) {
    grunt.fail.warn( 'Destination (' + f.dest + ') not written because src files were empty.' );
}
```

### PR DESCRIPTION
**Pull request by kfiku**
Fix bug with specifing url as src + test and update README.mc

### CODE REVIEW

File: tasks/uncss.js [hunk 1]
```diff
@@ -34,17 +34,17 @@ module.exports = function ( grunt ) {
                 }
             });
 
-            if ( src.length === 0 && file.src.length === 0 ) {
-                grunt.fail.warn( 'Destination (' + file.dest + ') not written because src files were empty.' );
-            }
-
-            file.src.forEach(function (source) {
+            file.orig.src.forEach(function (source) {
```
Comment (tobek) [inline tasks/uncss.js]: Getting `file.orig.src.forEach is not a function` here, fix in my PR against your repo https://github.com/kfiku/grunt-uncss/pull/1

**Comment by XhmikosR** [comment]
Landed in master, thanks!

**Comment by JohnAlbin** [comment]
I've tested this and it works great. Took me a while to realize why `nonull: true` was required for this feature, but it absolutely is. (I'm rather new-ish to Grunt.)

In fact, my build fails without this fix, so I'm now using @kfiku's fork until this PR is merged.

**Comment by JohnAlbin** [comment]
lol. I guess I should have refreshed this browser tab that I opened yesterday. Thanks @XhmikosR and @kfiku!

**Comment by XhmikosR** [comment]
You can use the tag until @addyosmani publishes v0.5.2 on npm. I don't have the needed rights on npm to do it myself :/

**Pull request closed without merging**